src/platform/BUILD.gn

```diff
@@ -197,13 +197,7 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
       sources += [
         "Darwin/BLEManagerImpl.cpp",
```

Comment:
should we also take out the BleManagerImpl.cpp (I believe it is currently one large #ifdef ?